### PR TITLE
Add headers parameter

### PR DESCRIPTION
**Pull request by jdbarillas**
I added a parameter that takes a character vector to be used as column names for the htmlTable. I also added a check to make sure that the length of that vector is equal to that of the table.

An added side effect is that you can add html within the character vector which will be reflected in the htmlTable output.

```
old_df = data.frame(var1 = c("A", "B", "C"),
                    val1 = c(1, 2, 3))
new_df = data.frame(var1 = c("A", "B", "C"),
                    val1 = c(1, 2, 4))
ctable = compare_df(new_df, old_df, c("var1"), 
                    headers = c("<i>varA</i>", "<span style='color:red;'>valB</span>"))
print(ctable$comparison_df)
ctable$html_output
```

Please feel free to critique, I don't have much experience with contributing.

### CODE REVIEW

**Comment by alexsanjoseph** [comment]
Thanks for the PR, the commit looks good. However, the build seems to be failing? And I'd add a small test as well.

Currently I'm a bit strapped for time, so if you can get to fixing ^, please go ahead. Else I'll work on it within a week or so 

Thanks again,
Alex

File: R/fnsComparison.R [hunk 1]
```diff
@@ -69,18 +71,31 @@ compare_df <- function(df_new, df_old, group_col, exclude = NULL, limit_html = 1
   if(nrow(comparison_table) == 0) stop_or_warn("The two data frames are the same after accounting for tolerance!", stop_on_error)
   if(nrow(comparison_table_diff) == 0) stop_or_warn("The two data frames are the same after accounting for tolerance!", stop_on_error)
 
-  if (limit_html > 0 & nrow(comparison_table_diff) > 0 & nrow(comparison_table) > 0)
-    html_table = create_html_table(comparison_table_diff, comparison_table_ts2char, group_col, limit_html, color_scheme) else
+  if (is.null(headers)) {
```
Comment (alexsanjoseph) [inline R/fnsComparison.R]: Also refactor the whole of if loop into a function?

get_headers_for_html_table <- function(headers){
...
}

**Comment by jdbarillas** [comment]
I turned the addition of the headers into the function and also created some tests for it. I believe the builds are failing due to packages. I wonder if it has something to do with me working from Linux. 

Thank you again for the feedback, it's been extremely helpful.

**Comment by alexsanjoseph** [comment]
Excellent PR from someone who claims to have little experience with contributing. I've made a few changes in implementation here. Please take a look. If it's alright with you I shall merge.

- Different implementation of headers which allows explicit naming of headers rather to prevent issues while ordering. 
- Also added explicit group column.
- Added more tests and edge cases

https://github.com/alexsanjoseph/compareDF/pull/16

**Comment by jdbarillas** [comment]
Thank you for the compliment and feedback! I learned some new things going through your changes, and I'm alright with merging the PR.
